### DISCUSSION TITLE
Whitelist breakable blocks in SurvivalRequiringClaims mode

### DISCUSSION THREAD

**Discussion by ShaneBeee** [GriefPrevention/GriefPrevention#197] [Ideas]
I like to use SurvivalRequiringClaims in my claims: mode: for my survival world, stopping players from destroying the world.
It would be cool if you had a section for maybe white listed blocks!
I would like to have certain blocks be breakable.
For example, I use ExtraHardMode, and when a player kills a spider, they can't break the cobweb around them that is placed. I would like to make that breakable, and maybe a few other things!

**Comment by RoboMWM** [comment]
Hmm, perhaps try adventure mode? Otherwise, an addon can implement this, and probably more easily so whenever I finish version 20.

**Comment by ShaneBeee** [comment]
Duly noted! I really wish I knew how to write add-ons ;)

**Comment by ShaneBeee** [comment]
I may look to find someone to write add-ons.
Basically I want players to be able to break most blocks, but not trees/grass, thus keeping the regular world looking nice!
